src/pages/resources/recipes/document-changes/index.md

```diff
@@ -21,6 +21,7 @@ The following script fragment shows how to create a new document and set the mar
 
 ```js 
 //Set the measurement units and ruler origin.
+const { MeasurementUnits, RulerOrigin} =  require("indesign");
```

Comment:
inconsistent code indentation. Should have a space before closing brackets.  Same issue in other places.